src/ck-core/ckrdma.C

```diff
@@ -1277,7 +1266,7 @@ void CkRdmaEMBcastAckHandler(void *ack) {
         } else {
           // Set zcMsgType to CMK_ZC_BCAST_RECV_ALL_DONE_MSG to signal to charmxi
           // that this is the final message containing the posted pointers
-          CMI_ZC_MSGTYPE(env) = CMK_ZC_BCAST_RECV_ALL_DONE_MSG;
+          CMI_ZC_MSGTYPE(myMsg) = CMK_ZC_BCAST_RECV_ALL_DONE_MSG;
```

Comment:
This was another minor bug fix that I made in the commit (unrelated to this PR). I'm still not sure why any test/example didn't catch it.